README.md

```diff
@@ -429,64 +438,80 @@ if (!global.BROWSER) {
 
 **Shape**
 ```js
-Module.appConfig = {
-  provideStateConfig: {
-    server: {
-      [settingName]: {
-        [environmentLevel]: String,
+if (!global.BROWSER) {
+  Module.appConfig = {
+    provideStateConfig: {
+      server: {
+        [settingName]: {
+          [environmentLevel]: String,
+        },
       },
-    },
-    client: {
-      [settingName]: {
-        [environmentLevel]: String,
+      client: {
+        [settingName]: {
+          [environmentLevel]: String,
+        },
       },
     },
-  },
-};
+  };
+}
 ```
 
 The `provideStateConfig` directive is useful for supplying string-based key value settings per runtime (e.g. `client` or `server`) and per `environmentLevel` (e.g. QA, Prod, etc).
```

Comment:
Mention `ONE_CONFIG_ENV`

Reply:
Fixed in 5e79f5aa8e38c38c0ca6db4a686a9868169d7631